### ISSUE TITLE
Lose of gradient in log_probs

### ISSUE THREAD

**Issue by BladeRunner2050** [Quantum-Cheese/DeepReinforcementLearning_Pytorch#1]
Hi, 
Actor_Critic/A3C/worker.py line 49:
`log_probs = torch.tensor([t[1] for t in self.memory], dtype=torch.float)`
I think this will lose the gradient of log_probs, is it right?
